src/sentry/monitors/consumers/check_in.py

```diff
@@ -90,14 +90,19 @@ def _process_message(wrapper: Dict) -> None:
 
     ratelimit_key = f"{params['monitor_slug']}:{environment}"
 
+    metric_kwargs = {
+        "source": "consumer",
+        "source_sdk": params.get("sdk", {}).get("name", "unknown"),
```

Comment:
Do we use this in metrics elsewhere? A bit worried about the cardinality here since this is just an arbitrary string passed along (i think?)

Reply:
I'm not too worried about the cardinality, since the SDKs set this, so we it should stay pretty low.

I checked with @JoshFerge and it sounds like they were planning to do the same thing

Not sure what we do elsewhere

Reply:
just to confirm, we won't send it with the metrics but the `version` is available too?

Reply:
Correct this is under the `sdk.version` key

Reply:
we weren't planning to do the same thing but regardless this is fine

Reply:
oh sorry, misinterpreted our conversation lol